src/DssSpellCollateralOnboarding.sol

```diff
@@ -29,71 +29,58 @@ contract DssSpellCollateralOnboardingAction {
     // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
     //
     // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmPgPVrVxDCGyNR5rGp9JC5AUxppLzUAqvncRJDcxQnX1u
+    //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
```

Comment:
Why was this updated?

Reply:
Checking with CES, it does not seem mandatory as the number that is used (0.75%) was already there.

Reply:
It got changed because I copied the Goerli spell over, that apparently uses a different IPFS link in the template. I wasn't aware they were different - feel free to revert back to the original one.